src/modules/45-dashboards/components/DashboardList/DashboardList.tsx

```diff
@@ -36,6 +35,15 @@ const DashboardList: React.FC<DashboardListProps> = ({
 }): React.ReactElement => {
   const { getString } = useStrings()
   const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
+  const { editableFolders } = useDashboardsContext()
+
+  const isCloneable: boolean = React.useMemo(() => {
+    return editableFolders ? editableFolders.length > 0 : false
+  }, [editableFolders])
```

Comment:
Same here... the check to figure out if it's cloneable is less work than the memoisation and lookup, might as well inline.
```suggestion
  const isCloneable = !!editableFolders.length
```